cmd/nvidia-ctk-installer/toolkit/installer/artifact-root.go

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
is there a reason this license header is different?

Reply:
Different from what? The reason is that these changes have been on a branch associated with #602 for some time and I am factoring them out. Will update the header.